epl-nft/contracts/EnglishPremierLeague.cdc

```diff
@@ -802,6 +800,12 @@ pub contract EnglishPremierLeague: NonFungibleToken {
             return <- momentNFT
         }
 
+        /// Purchase Pack
+        ///
+        pub fun purchasePack(purchaseAddress: Address, packID: UInt64) {
```

Comment:
How it is working? In actuality how the pack is going to deliver to the user?

Reply:
EPL is following a bit of a different model. Our packs are offchain and will be delivered and opened by the wallet on purchase. The moments & off-chain packs are mint-on-demand and always on. There is an edge case where the FCL tx to purchase succeeds and then disconnects before fulfilling their pack. This event is our solution to reconcile lost pack purchases when this occurs. @javiersoto15 did this diagram illustrating the flow.
![recatch](https://user-images.githubusercontent.com/896651/230448454-f3294f33-f491-4032-a851-112ea748bf8f.png)